### ISSUE TITLE
Dataset GoogleDrive expired

### ISSUE THREAD

**Issue by TimSong412** [bmild/nerf#214]
Hi! It seems like the google drive link for the dataset is expired or removed. May I know how I can get the training data now?

**Comment by saisharathsadu** [comment]
Yes. Even I am looking for the same. Share the data if anyone has it?

**Comment by TimSong412** [comment]
just got the synthesized blender here. personal used
https://drive.google.com/file/d/1ZzYxo5v_C76QmbyTAK1jChxtQMeLRUXK/view?usp=sharing

**Comment by saisharathsadu** [comment]
Thanks for sharing!

**Comment by wuyou012** [comment]
> Hi! It seems like the google drive link for the dataset is expired or removed. May I know how I can get the training data now?

Here is nerf_llff_data you can use:
https://drive.google.com/drive/u/1/folders/1PXEDEECbxhEFG_PkMekJuqVRZB4roDR_

**Comment by Hamme122** [comment]
> > Hi! It seems like the google drive link for the dataset is expired or removed. May I know how I can get the training data now?
> 
> Here is nerf_llff_data you can use: https://drive.google.com/drive/u/1/folders/1PXEDEECbxhEFG_PkMekJuqVRZB4roDR_

Hi wuyou, this link seems to be invalid, can you share a new one! Thanks!

**Comment by Hamme122** [comment]
> > > Hi! It seems like the google drive link for the dataset is expired or removed. May I know how I can get the training data now?
> > 
> > 
> > Here is nerf_llff_data you can use: https://drive.google.com/drive/u/1/folders/1PXEDEECbxhEFG_PkMekJuqVRZB4roDR_
> 
> Hi wuyou, this link seems to be invalid, can you share a new one! Thanks!

Dataset in Kaggle is available: https://www.kaggle.com/datasets/arenagrenade/llff-dataset-full/data.

**Comment by wuyou012** [comment]
> > > Hi! It seems like the google drive link for the dataset is expired or removed. May I know how I can get the training data now?
> > 
> > 
> > Here is nerf_llff_data you can use: https://drive.google.com/drive/u/1/folders/1PXEDEECbxhEFG_PkMekJuqVRZB4roDR_
> 
> Hi wuyou, this link seems to be invalid, can you share a new one! Thanks!

Here is the newest one, with llff and synthetic: https://drive.google.com/drive/folders/1PXEDEECbxhEFG_PkMekJuqVRZB4roDR_?usp=drive_link
